xblock/exceptions.py

```diff
@@ -30,8 +30,9 @@ def __init__(self, saved_fields, dirty_fields):
         `dirty_fields` - a set of fields that were left dirty after the save
         """
         # Exception is an old-style class, so can't use super
-        Exception.__init__(self)
+        Exception.__init__(self, message)
```

Comment:
Oh, I thought i added the comment earlier, We are setting the message attribute to the `Exception`
E.g.
```
try:
    1/0
except Exception as e:
    log(e.message)
```
So we have to set the message class attribute.